provider/database.py

```diff
@@ -18,7 +18,8 @@
 from cloudant.result import Result
 
 class Database:
-    dbname = "ow_kafka_triggers"
+    db_prefix = os.environ['DB_PREFIX']
+    dbname = db_prefix + 'ow_kafka_triggers'
```

Comment:
My thought process here is that the user may not want to create a database called "ow_kafka_triggers."